### ISSUE TITLE
Questions and usage

### ISSUE THREAD

**Issue by InfiniteWorldLegacy** [FlagBrew/PKSM#1092]
Hello, I have a few questions, and a minor issue regarding PKSM
I have updated to the latest version yesterday, last time I updated was back in 2018.
1. Minor problem: ''Note: If loading slowly, try deleting some old backups! This helped a team member go from an about 10 second load time to 2''
So uh... regarding that, it does take 10 seconds to load, but what kind o backup do we need to delete? i've deleted everything in the PKSM folder(sd:3DS/PKSM/everything deleted) before updating to the latest version, but loading still takes more than 2 seconds. (minnor problem anyways)

2. Also, I use Twilight Menu++ to play NDS games, but I have no Idea on how to make PKSM search the path until my .nds save file.
I've read the topic
https://github.com/FlagBrew/PKSM/issues/1087
and this one too
https://github.com/FlagBrew/PKSM/wiki/Configuration#extra-saves
How do I make PSKM follow that directory? I see no option when i'm in PSKM main screen, can anyone make a step by step? I have also the MANUAL but it doesn't show me clearly what I needed to do
I wanted a tutorial on how to use Extra save files...

3. GPSS, a cloud-based Pokémon sharing service! ''You can also upload Pokémon from your PC.''
Is this talking about my real life PC or the in-game PC? Because if it is the real life PC I have no idea on how to make that happen, whennever I Tap the wireless Icon on Storage, it shows every pokemon on the GPSS and I press X and asks ''Do you want to enter a share code?'' And that's it, I have no idea of what it really means. Plus, if it says ''This is the only planned servepkx replacement.'' how do I use it? Back in the days, we were able to send .pkm files from our real pc to the pksm box storage...

I hope you guys can help me, tks

**Comment by GriffinG1** [comment]
![image](https://user-images.githubusercontent.com/28538707/71562139-02b45b00-2a3c-11ea-9deb-da390de74b8d.png)
This should cover your extrasaves question. @piepie62 pls update wiki

Boot times are still gonna be a few seconds. PKSM still has to load in the bank, check for updates, etc.

GPSS lets you upload pokemon to and from a wireless server to share with others. The PC accessible site is [here](https://flagbrew.org/gpss). Just select a pokemon in the gpss and you can download it, or drag and drop one of yours into the gpss and you can upload it. The share code is referring to if you find them through the site. They each have their own code you can enter to retrieve them that way, as well as a qr code you can scan from that same code enter screen. 

Uploading from PC doesn't have an interface currently. @FM1337 maybe add some stuff to the wiki on uploading from PC?

**Comment by piepie62** [comment]
It's in the configuration section, seems obvious to me that you need to be in the configuration screen to access it

**Comment by GriffinG1** [comment]
Not for someone using pksm for the first time who doesn’t know how to get to configuration from titles

**Comment by InfiniteWorldLegacy** [comment]
yeah, loading is just a minor, as long as it loads and doesn't freeze it's perfect :)
Thank you @GriffinG1 It helped me a lot, I was finally able to make use of the Extra files, these steps were a perfeclty explanation. It worked like a charm for my HeartGold Save file, but it didn't work for my diamond and pearl(Kept saying that the format was invalid) but i believe it's because I didn't convert the save file from .dsv to .sav when I was using DeSmuME, I just renamed to .sav because TwilightMenu++ can read it with no problems. As for my HeartGold, I probably converted to .sav and that's why it worked. I will check that later today and confirm it

Regarding GPSS, thank you so much once again, so as for the time being, we can only upload/drag and drop one of our pokemon from our current in-game pc. Ok, understood. And thank you for the clarification on the share code as well.

@piepie62  Hi, I'm sorry but it wasn't very obvious yet even though I used PKSM in the past, I have never tried to use the extra saves, so it was pretty much like using pksm for the first time. I mean, the instructions were always saying ''it's possible to do this, these and that'' but it does not say ''its possible by pressing this, selecting these and those'' so, for a newbie like me, I didn't understand what I had to do, sorry. So the Printscreen shared by @GriffinG1 made a lot more sense to me

Anyways, thank you both for developing such an amazing tool
